src/components/display-card/helper/csv_helper.js

```diff
@@ -0,0 +1,25 @@
+const csvHelper = (cards, choices) => {
+    const headers = [
+        "id",
```

Comment:
Can we make all these variable names in Pascal Case so that it is more readable?

Reply:
done